examples/chip-tool/commands/common/Commands.cpp

```diff
@@ -54,6 +55,22 @@ int Commands::Run(int argc, char ** argv)
     Command * command = nullptr;
     NodeId localId;
     NodeId remoteId;
+    char * argNodeId = nullptr;
+
+    const char * optstring = "n::";
+    int opt;
+    while ((opt = getopt(argc, argv, optstring)) != -1)
```

Comment:
The help seems to have been placed for 'showclusters' however this runs on every command. Is this correct?

Chip has a CHIPArgParser.hpp that seems to be used for argument parsing, would be nice to standardize on it